packages/data-explorer/src/main.ts

```diff
@@ -74,18 +74,32 @@ Vue.config.productionTip = false
 // This way the mainView can coordinate the events
 Vue.prototype.$eventBus = new Vue()
 
+// @ts-ignore
+const { lng, fallbackLng } = window.__INITIAL_STATE__
```

Comment:
Maybe add this line before line #98 "Vue.use" to group it together?

Reply:
I removed them together with the __INITIAL__STATE in index.html